src/ios/main.mm

```diff
@@ -270,6 +266,12 @@ -           (BOOL) application: (UIApplication*) application
   NSURL* url;
   int port = getDevPort();
 
+  if (@available(iOS 16.4, *)) {
```

Comment:
we can add tvOS too for future 😎

Reply:
haha yeah we can come back to that - probably will want to have a `src/tvos/main.mm` file